Student Standing Feature (#1468)
pupilfirst/pupilfirst: docs/users/school_standing.md

3. **Add/Edit/View Standings:** You can add, edit, and view the standings for your school. You can add a new standing by clicking on the `Add another standing` button. You can edit a standing by clicking on the `Edit` icon next to the standing. You can delete a standing by clicking on the `Delete` icon on the standing card.

![Add/Edit Standing Form](../assets/school_standing/standing_edit_page.png)

> **Note:** When you enable the standing for your school for the first time, a default standing will be created for you. You can edit the default standing, but you cannot delete it. The default standing is used as the initial standing for all the members of your school.

The standing creation and edit form has the following fields:

- **Name:** The name of the standing. For example, `Neutral`, `Good`, `Community Hero`, `Warned`, `Disreputed` etc.

> - **Description:** The description of the standing. It is optional, and can be used to describe the standing in a sentence. This description will be shown to admins (in the student standing editor), students (in their standing page) and to organisation admins (when viewing student records).

Reviewer: ```suggestion
- **Description:** The description of the standing. It is optional, but we recommend that you use it to briefly describe the standing. This description will be shown to admins (in the student standing editor), students (in their standing page) and to organisation admins (when viewing student records).
```
